app/src/main/java/com/example/countryinfo/CountryDetails.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.countryinfo
+
+class CountryDetails(
+    val name: String,
+    val capital: String,
+    val region: String,
+    val population: Double,
+    val flag: String,
+    val currencyNames: List<String>,
+    val currencySymbols: List<String>,
```

Comment:
Do we actually need currency symbols on UI. I was under impression that we agreed to just use currency names?

Reply:
Sorry, forgot about that. Removed symbols.